src/components/widgets/IssueCredential.tsx

```diff
@@ -101,15 +102,26 @@ const IssueCredential: React.FC<BarChartProps> = ({
 
   const sendVC = async (body: any) => {
     try {
-      await agent?.sendMessageDIDCommAlpha1({
-        data: {
-          to: subject as string,
-          from: issuer as string,
-          type: proofFormat,
-          body: proofFormat === 'jwt' ? body.proof.jwt : body,
-        },
-        save: true,
+      const messageId = uuidv4()
+      const message = {
+        type: 'application/didcomm-encrypted+json',
```

Comment:
@awoie does this conflict with message type `veramo.io-chat-v1` ?

Reply:
> @awoie does this conflict with message type `veramo.io-chat-v1` ?

IMO, the packer should set the `typ` (mime-type) of the packed DIDComm message, e.g., protected header in JWE (for authcrypt). So, in that case, you should not set `application/didcomm-encrypted+json`. Also note, there is a difference between `typ` and `type`. `type` is the right one for veramo.io-chat-v1 although I'd use something like this: `veramo.io/chat/v1/basicmessage`

Reply:
ok. It's the above correct for a standard non-chat message?

Reply:
@gbugyis ^

Reply:
@jasonhealy doing this `type: 'application/didcomm-encrypted+json' is not correct. You'd rather set the protocol message type there, e.g., `veramo.io/chat/v1/basicmessage`.  `application/didcomm-encrypted+json` is automatically set by the packer.

Reply:
Gotcha. I'll update it.